tensorflow/core/platform/env.h

```diff
@@ -381,6 +385,10 @@ Status ReadFileToString(Env* env, const string& fname, string* data);
 Status WriteStringToFile(Env* env, const string& fname,
                          const StringPiece& data);
 
+/// A utility routine: copy contents of `oldpath` to `newpath`
+Status CopyFile(Env* env, const string& oldpath, const string& newpath,
```

Comment:
I don't see where this is implemented?

Reply:
Sorry, that was left over when check in. It has been cleaned.